src/logic/EligibilityTimelineCalculator.js

```diff
@@ -23,9 +23,18 @@ export default class EligibilityTimelineCalculator {
         let lastConvictionDateString = lastConvictionDate.format('YYYY-MM-DD');
         let yearsSinceLastConvictionDate = calculationDate.diff(lastConvictionDate, 'years');
 
+
         input.convictions.forEach((conviction) => {
             let convictionOutput = output.getConviction(conviction.id);
 
+            // Check to see if the conviction is DUI-related.
```

Comment:
I'm not a fan of comments unless they are really necessary. Your code in lines 32-34 seems already expressive and shows intent. The comments seem redundant.